src/ripple/nodestore/impl/DatabaseImp.h

```diff
@@ -225,8 +220,12 @@ class DatabaseImp
         {
             // Yes so at last we will try the main database.
             //
+            auto const before = std::chrono::steady_clock::now();
```

Comment:
Mark, the difference is that a `steady_clock` never gets adjusted.  So it's timepoint values can never go backwards.  A `high_resolution_clock`gives the shortest tick period possible.  But a `high_resolution_clock` is not guaranteed to be steady.  You decide which characteristic is more important for your measurements.